File: Solutions/beta/beta_which_operator.py
def whichOper(a, b, oper):
    return {'a':lambda x,y: x+y,
            's':lambda x,y: x-y,
            'm':lambda x,y: x*y,
            'd':lambda x,y: x/y}[oper[0]](a,b)

File: Solutions/beta/beta_which_operator.js
function whichOper(num1, num2, op){
  ops = {'a': function add (a,b) {return a+b},
         's': function sub (a,b) {return a-b},
         'm': function mul (a,b) {return a*b},
         'd': function div (a,b) {return a/b}
         }
  return ops[op[0]](num1,num2)      
}
